dbt/adapters/athena/impl.py

```diff
@@ -110,32 +109,12 @@ def clean_up_partitions(self, database_name: str, table_name: str, where_conditi
         }
         partition_pg = paginator.paginate(**partition_params)
         partitions = partition_pg.build_full_result().get("Partitions")
-        s3_rg = re.compile("s3://([^/]*)/(.*)")
         for partition in partitions:
             logger.debug(
```

Comment:
Can you move this logger in `_delete_from_s3` so the logging is compliant with the actual path being deleted. It will help a lot for debugging (i.e. we are sure that this is path used and therefore no transformation made afterward).

Reply:
LGTM that way ✅